index.js

```diff
@@ -251,16 +266,15 @@ module.exports = function (schema, options) {
         return this.save(cb);
     };
 
-    schema.statics.delete =  function (conditions, deletedBy, callback) {
-        if (typeof deletedBy === 'function') {
-            callback = deletedBy;
-            conditions = conditions;
-            deletedBy = null;
+    schema.statics.delete =  function (conditions, params = {}, callback) {
+        if (typeof params === 'function') {
+            callback = params;
```

Comment:
🤔 I'm thinking that you need to have params defined as an `{}` after this point, so try adding `params = {}` but you'll want to double check that this doesn't go and set `callback` to the same `{}`